src/tracks/InstrumentTrack.cpp

```diff
@@ -1130,6 +1156,37 @@ void InstrumentTrackView::midiConfigChanged()
 
 
 
+QMenu * InstrumentTrackView::createFxMenu(QString title, QString newFxLabel)
+{
+	int channelIndex = model()->effectChannelModel()->value();
+
+	FxChannel *fxChannel = Engine::fxMixer()->effectChannel( channelIndex );
+
+	QMenu *fxMenu = new QMenu( title.arg( channelIndex ).arg( fxChannel->m_name ) );
```

Comment:
I am having trouble here as I want the marker to be optional.

![2015-01-15-114643_881x263_scrot](https://cloud.githubusercontent.com/assets/347552/5758493/3dd655de-9cad-11e4-9366-f76cc2238065.png)

I could not find any reference on how to ignore these warnings, if this persists I will check if they appear on the release build.